clinica/lib/nipype/interfaces/fsl/epi.py

```diff
@@ -0,0 +1,342 @@
+# -*- coding: utf-8 -*-
```

Comment:
Sounds to me that upstream nipype could benefit from this interface definition. What do you think?

Reply:
Actually what's different from this:

https://github.com/nipy/nipype/blob/7080ef9edd0fe095323a90b0b77be7a0b73824a5/nipype/interfaces/fsl/epi.py#L899

Reply:
When the interface was modified and added to Clinica, it was for 2 purposes:
- Add the `initrand ` option (now present in newest version of Nipype, only the help message seems to be different)
- Choose between `eddy_cuda8.0` and `eddy_cuda9.1`. I believe this was added in FSL 6.X. Honestly, I don't know if this modification was neat or not.

If you have some time, I would like your feedback regarding the handling of `eddy_cuda*` :) This would be ultimately into Nipype.

Reply:
The upstream [Eddy interface](https://github.com/nipy/nipype/blob/7080ef9edd0fe095323a90b0b77be7a0b73824a5/nipype/interfaces/fsl/epi.py#L899) only references `eddy_cuda`. What's the use for the additional `eddy_cuda${CUDA_VERSION}`?

Reply:
The official [documentation](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/eddy/UsersGuide#The_eddy_executables) states that either the user is provided with an FSL distribution with `eddy_cuda` already setup or the user is responsible for creating the appropriate symlink.

Based on that, I don't think exposing `eddy_cuda${CUDA_VERSION}` to the nipype interface is the way forward. Instead we should be using the upstream Eddy nipype interface and ensure the appropriate symlink is present in our clusters.

Reply:
Though the last update is from `25-09-2019`, I never noticed this piece of information, thank you so much for this feedback :)

Regarding the documentation and implementation, I will in that case add the `eddy_cuda` option, remove `eddy_cuda{8.0|9.1}`  and give explanations/link to the documentation you gave me, that should do the job. Would that suit you @ghisvail?

Reply:
Why not just drop this code altogether and use the upstream nipype Eddy interface then?

Reply:
Sorry, this what I meant: after using the Nipype interface, I will remove `eddy_cuda{8.0|9.1}`  option in the pipeline & CLI and replace by `eddy_cuda` option.

Reply:
great